Picture.py

```diff
@@ -358,14 +435,19 @@ def OnInit(self):
                 self.frame.Show()
                 return True
 
-        # Need wx app to run in separate thread or process.  It seems
-        # that starting a wxPython GUI in a worker thread is not a good idea
-        # so the easiest thing to do is just fork a new process.
-        pid = os.fork()
-        if pid == 0:
+        def doShow():
+            """Run the wx app to show the image"""
             wxImage = self.getWxImage()
-            app = ShowImage(image=wxImage, title=self.fileName)
+            app = ShowImage(image=wxImage, title=self.title)
             app.MainLoop()
-            exit(0)
+            #exit(0)
+
+        # We want control to return immediately to the command prompt or
+        # calling script but wx will block because of wx.MainLoop().  To
+        # get around this we start a new process to run the wx GUI.
+
+        p = Process(target=doShow, args=())
+        p.start()
```

Comment:
Upon further testing whatever it is that you have done here in picture has caused show to not work properly.(It gives an EOFError: Ran out of input error when trying to use show(pic))

Reply:
yep - this doesn't work on Windows